src/Components/Feature/Redux/ToDoReducer.tsx

```diff
@@ -0,0 +1,31 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { IToDoResp } from '../../../Shared/types';
+
+interface TodosState {
+  todos: IToDoResp[];
+}
+
+const initialState: TodosState = {
+  todos: [],
+};
+
+const todosSlice = createSlice({
```

Comment:
Can you please check RTK query doc how we can create slice without managing state. Consider this is an enhancement